### ISSUE TITLE
test failure

### ISSUE THREAD

**Issue by bergmark** [myfreeweb/microformats2-parser#5]
Using microformats2-parser

```
Failures:

  test-suite/Data/Microformats2/Parser/PropertySpec.hs:45:7:
  1) Data.Microformats2.Parser.Property.extractProperty parses u- properties
       expected: Just ("https://secure.gravatar.com/avatar/947b5f3f323da0ef785b6f02d9c265d6?s=96&d=blank&r=g",True)
        but got: Just ("https://secure.gravatar.com/avatar/947b5f3f323da0ef785b6f02d9c265d6?s=96",True)
```

I couldn't tell why this started failing, but it may have to do with some new dependency, this is the diff in stackage nightly for before and after the test suite was marked as an expected failure:

https://www.stackage.org/diff/nightly-2018-06-23/nightly-2018-06-24

**Comment by myfreeweb** [comment]
ah, html-conduit 1.3.1 with an "entity decoding in attribute value bug fix". Created issue: https://github.com/snoyberg/xml/issues/128

**Comment by myfreeweb** [comment]
Released 1.0.1.8 with this fixed.
